### ISSUE TITLE
Training converges to a single move

### ISSUE THREAD

**Issue by lunathanael** [bwfbowen/muax#11]
Hello, I came across this project and decided to use it for my RL project.
I'm facing an issue where after the initial training step, the model only predicts one move. Looking into the initial predictions, it seems the Pi logits is not uniform distributed. I'm wondering if for environments with a larger action space, one should produce initial data using a uniform network.

I'm using the EZ functions and gumbel muzero.
